trillian/ctfe/handlers.go

```diff
@@ -624,6 +637,12 @@ func getEntries(ctx context.Context, c *LogContext, w http.ResponseWriter, r *ht
 		if err != nil {
 			return c.toHTTPStatus(err), fmt.Errorf("backend GetLeavesByRange request failed: %v", err)
 		}
+		if rsp.GetSignedLogRoot() != nil && rsp.GetSignedLogRoot().TreeSize <= start {
+			// If the returned tree is too small to contain any leaves return the 4xx
+			// explicitly here. It was previously returned via the error status
```

Comment:
Second sentence will go out of date -- just drop? (passim)

Reply:
Done.